falkordb-cluster/cluster-entrypoint.sh

```
@@ -328,6 +350,28 @@
   redis_exporter_pid=$!
 fi
 
+meet_unkown_nodes(){
+  # Look for nodes that have 0@0 in the nodes.conf and meet them again"
+  if [[ -f "$DATA_DIR/nodes.conf" && -s "$DATA_DIR/nodes.conf" ]];then
+    discrepancy=0
+    while IFX= read -r line;do
```

Shellcheck (0.10.0)

[warning] Remove space after = if trying to assign a value (for empty string, use var='' ... ).

(SC1007)